client/packages/common/src/intl/locales/en/coldchain.json

```diff
@@ -26,6 +26,7 @@
   "error.code-no-match": "{{field}} not found in catalogue",
   "error.unable-to-scan": "Unable to scan QR code",
   "error.duplicate-asset-number": "duplicate asset number in csv",
+  "error.invalid-field-value": "{{ value }} isn't a valid {{ field }}",
```

Comment:
Is it ok for coldchain translations to be used in catalogue, or should I move some things around?